web3-actions/priceFeedMonitor/tokenPairMonitor/package.json

```
@@ -0,0 +1,15 @@
+{
+    "name": "tokenPairMonitor",
+    "scripts": {
+        "build": "tsc"
+    },
+    "devDependencies": {
+        "typescript": "^4.3.5"
+    },
+    "dependencies": {
+        "@tenderly/actions": "^0.2.0",
+        "@types/node": "^20.8.10",
```

Please re-create the package-lock.json after adding the following dependency 🙏 
```suggestion
        "@types/node": "^20.8.10",
        "ethers": "^5.7.2",
```